src/EditorFeatures/Core.Wpf/InlineRename/UI/SmartRename/SmartRenameViewModel.cs

```diff
@@ -150,7 +153,39 @@ private void OnGetSuggestionsCommandExecute()
                 _suggestionsDropdownTelemetry.DropdownButtonClickTimes += 1;
             }
 
-            _getSuggestionsTask = _smartRenameSession.GetSuggestionsAsync(_cancellationTokenSource.Token).CompletesAsyncOperation(listenerToken);
+            var allRenameLocations = BaseViewModel.Session.AllRenameLocationsTask.Join();
+            ImmutableDictionary<string, string[]> context = default;
```

Comment:
we prefer no arrays.  can you make this an `ImmutableArray<string>`?

Reply:
good to know, done!